.github/workflows/autobuild.yml

```diff
@@ -80,42 +73,40 @@ jobs:
              draft:                 false
 
 
-         ### CANCEL ### can be used for development concerning release-creation
-         #- name:                    Cancelthrougherroe
-         #  run:                     myundefinedfunction
-
-
   release_assets:
     name:                           Build assets for ${{ matrix.config.config_name }}
```

Comment:
```suggestion
    name:                           Build for ${{ matrix.config.config_name }}
```

Reply:
Github-wise the term is `assets`, so if we want to provide details, we should probably keep this as-is?

Reply:
Yes, but I'm unsure if it adds any value.

Reply:
Ok, removed. It also matches the first job's name better, now that the imperative has been dropped.